tests/templates/kuttl/aas-user-info/02-install-aas.yaml.j2

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - << EOF
```

Comment:
Sorry for the stupid question, but is the `kubectl apply` needed here? At first glance it looks like we can just use normal YAML objects

Reply:
I can take a look, but I copy pasted this from the keycloak test: https://github.com/stackabletech/opa-operator/blob/main/tests/templates/kuttl/keycloak-user-info/02-install-keycloak.yaml.j2

Reply:
https://github.com/stackabletech/opa-operator/pull/519/commits/1eda08ff005101cf037a58a12f2c40fc5633827e